app/client/websocket/index.ts

```diff
@@ -79,8 +79,12 @@ export default class WebSocketClient {
             return;
         }
 
-        const config = await getConfig(database);
-        const connectionUrl = (config.WebsocketURL || this.serverUrl) + '/api/v4/websocket';
+        const [websocketUrl, version, reliableWebsocketConfig] = await Promise.all([
```

Comment:
The way the config is now stored in the database. Instead of one single row in the database, is several rows. So getting all is doing some shenanigans to make it look like it was, and it is costly. This should be faster.